docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -0,0 +1,494 @@
+    Arn:               arn:aws:emr-containers:us-west-2:012345678910:/virtualclusters/dxnqujbxexzri28ph1wspbxo0/jobruns/000000030mrd934cdqc
+    Owner Account ID:  012345678910
+    Region:            us-west-2
+  Conditions:
+    Last Transition Time:  2022-08-26T18:29:12Z
+    Message:               Resource synced successfully
+    Reason:                
+    Status:                True
+    Type:                  ACK.ResourceSynced
+  Id:                      000000030mrd934cdqc
+Events:                    <none>
+```
+
+## Cleanup
+Simply run these commands to cleanup your environment
+```
+# delete all custom resources
+kubectl delete -f virtualcluster.yaml
+# note: you cannot delete jobruns unless you want to cancel them
+
+# uninstall emrcontainers controller
+helm delete ack-$SERVICE-controller -n $ACK_SYSTEM_NAMESPACE
+
+# delete crds
+kubectl delete crd/virtualclusters.emrcontainers.services.k8s.aws
+kubectl delete crd/jobruns.emrcontainers.services.k8s.aws
+
+# delete namespace
+kubectl delete ns $ACK_K8S_NAMESPACE
+kubectl delete ns $EMR_NAMESPACE
+
+# delete EKS cluster
+eksctl delete cluster --name "${EKS_CLUSTER_NAME}"
+```
+
+## Limitations
+
+1. JobRun CRD doesn’t have complete features available due to an issue with how ACK handles cyclic structure. Until this [issue](https://github.com/aws-controllers-k8s/community/issues/1445) is resolved, customers can’t use all the configuration from `**configuration overrides**` section.
+```
+# configurationOverrides:
+#   applicationConfiguration:
+#   monitoringConfiguration:
+#     cloudWatchMonitoringConfiguration:
+#       logGroupName:
+#       logStreamNamePrefix:
+#     s3MonitoringConfiguration:
+#       logUri:
+```
+2. You cannot delete a JobRun unless its in **error** state. There is no delete-job-run API for deleting jobs (for good reason). However, if your JobRun goes into error state, you can run `kubectl delete jobrun/<job-run-name>` to cancel the job.
+
+## Troubleshooting
+
+* If you run into issues creating VirtualCluster or JobRuns, check EMR on EKS controller logs for troubleshooting
+```
+CONTROLLER_POD=$(kubectl get pod -n ${ACK_K8S_NAMESPACE} -o jsonpath='{.items..metadata.name}')
+kubectl logs ${CONTROLLER_POD} -n ${ACK_K8S_NAMESPACE}
+```
+* You can enable debug logs for EMR on EKS controller if you are unable to determine cause of the error. You need to change values for `enable-development-logging` to `true` and `--log-level` to `debug`
+```
+CONTROLLER_DEPLOYMENT=$(kubectl get deploy -n ${ACK_K8S_NAMESPACE} -o jsonpath='{.items..metadata.name}')
+kubectl edit deploy/${CONTROLLER_DEPLOYMENT} -n ${ACK_K8S_NAMESPACE}
+```
+This is how your values should look after change.
+```
+        - --aws-region
+        - $(AWS_REGION)
+        - --aws-endpoint-url
+        - $(AWS_ENDPOINT_URL)
+        **- --enable-development-logging
+        - "true"
+        - --log-level
+        - debug**
+        - --resource-tags
+        - $(ACK_RESOURCE_TAGS)
+```
```

Comment:
Although this works, a better user experience would be set those values when starting/upgrading the helm chart, for example:

```bash
helm install --create-namespace -n $ACK_SYSTEM_NAMESPACE ack-$SERVICE-controller \
  oci://public.ecr.aws/aws-controllers-k8s/$SERVICE-chart --version=$RELEASE_VERSION \
  --set=aws.region=$AWS_REGION \
  --set=log.level=debug \
  --set=log.enable_development_logging=true
```

Reply:
Thanks @A-Hilaly for the feedback. I left this nugget here for folks who like to troubleshoot using kubectl. After all, these little nuggets makes these tutorials exciting :smiley: 
Do you see developers use helm for enabling debug while they troubleshoot? Let me know and I can add it as an additional option for users who use helm